### ISSUE TITLE
Run configuration forgets working directory

### ISSUE THREAD

**Issue by krojew** [intellij-rust/intellij-rust#6052]
## Environment

* **IntelliJ Rust plugin version:** 0.3.129.3308-202
* **Rust toolchain version:** 1.48
* **IDE name and version:** CLion 2020.2
* **Operating system:** Windows 10

## Problem description

After running a cargo run configuration with custom working directory, getting back to editing such configuration reverts the directory to workspace default.

## Steps to reproduce

1. Create a workspace with a binary project.
2. Create a run configuration and change the working directory from workspace root to project root.
3. Run the project - should work in changed working directory.
4. Edit run configuration again and notice the working directory reverted to workspace root.

**Comment by hydra** [comment]
Yeah, this is driving me nuts, wastes so much time having to re-specify the working directory.

This is pretty common when working on embedded software where you have multiple launch configurations with different features, i.e. when targetting different PCB revisions, MCUs, etc, and that each launch configuration runs cargo commands to make `.elf` or `.bin` files.

This is especially annoying as it's not always obvious that it's happening, e.g. if your crate with the binary uses a cargo.toml the settings in the cargo.toml will not be picked up from the crate, but instead will be picked up from the workspace root, resulting in things as odd as different linkers, optimization settings, etc being used and random build failures.

Another way this manifests is where your binary has `required` features, when the launch configuration is built from the correct crate directory that contains the bins, intellij will add the features automatically, but after it forgets the launch configuration directory you'll get the error message:

```
error: target `<bin>` in package `<crate>` requires the features: `...`, `...`, ...
```
